libraries/Microsoft.Bot.Connector/Authentication/SimpleChannelProvider.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Connector.Authentication
+{
+    public class SimpleChannelProvider : IChannelProvider
+    {
+        public string ChannelService { get; set; }
+
+        public SimpleChannelProvider()
+        {
+        }
+
+        public SimpleChannelProvider(string channelService)
+        {
+            this.ChannelService = channelService;
+        }
```

Comment:
null parameter check / throw on null

Reply:
No, null is an ok value. I added this in comments (and see above comment for rational).